### ISSUE TITLE
Empty ShapeList causes show_all to fail

### ISSUE THREAD

**Issue by MatthiasJ1** [bernhard-42/vscode-ocp-cad-viewer#90]
```py
from ocp_vscode import *
from build123d import *

x = ShapeList()

show_all()
```

```
show_all: list index out of range
```

**Comment by bernhard-42** [comment]
I am about to publish a version where the converter from build123d to OCP has been completely rewritten.
Great test case, already 
![image](https://github.com/user-attachments/assets/be4d44b8-5f7e-4d10-87d6-6cbe81a069a7)
works in new version

**Comment by bernhard-42** [comment]
fixed on 2.4.0
